api/src/main/java/org/openmrs/api/context/Context.java

```diff
@@ -890,6 +892,57 @@ public static void evictFromSession(Object obj) {
 		getContextDAO().evictFromSession(obj);
 	}
 
+	/**
+	 * Evicts the entity data for a particular entity instance.
+	 *
+	 * @param sessionFactory
+	 * @param entityClass entity class of the instance to evict from the DB cache
+	 * @param object entity instance to evict from the DB cache
+	 */
+	public static void evictSingleEntity(SessionFactory sessionFactory, Class<?> entityClass, OpenmrsObject object) {
```

Comment:
I would model this on how `evictFromSession()` above works. TBH, I'm not actually sure why we need this.

Reply:
Using ```evictFromSession()``` doesn't seem to do the trick of removing a single entity from cache as shown [here](https://drive.google.com/file/d/1fDt0VemSFk5CteqhdTlMZh1zBSWBo4E5/view?usp=sharing).